src/get-markdown-notes.js

```diff
@@ -1,27 +1,42 @@
 const fs = require("fs");
 const path = require("path");
 
+function toRegExp(value) {
+  if (typeof value === "string") {
+    return new RegExp(`^${value}$`);
+  }
+  return value;
+}
```

Comment:
This is pretty naive, only strings will get turned to `RegExps`, anything else is not changed. That means that if someone passes a value other than a `RegExp` or `String` to the `exclude` option, the build will fail. 

I figure if that becomes a concern, the best way to handle it would be to have a validation step for these arguments earlier in the library, rather than handling them at this low level 😀

Reply:
This seems fine for now. Yeah there is very little validation right now in general, and that may be something to work on, but this shouldn't be a big deal yet